docs/reference/8-admin-app/technical-articles/updating-admin-app-22-for-ed-fi-odsapi-cloud-deployment-for-azure-suite-3.md

```diff
@@ -0,0 +1,314 @@
+use [EdFi_Security]
+IF DATABASE_PRINCIPAL_ID('EdFiOdsAdminApp') IS NULL
+BEGIN
+    CREATE USER [EdFiOdsAdminApp] FOR LOGIN [EdFiOdsAdminApp] WITH DEFAULT_SCHEMA=[dbo]
+END
+--Assign roles to EdFiOdsAdminApp on EdFi_Security database
+EXEC sp_addrolemember 'db_datareader', 'EdFiOdsAdminApp'
+EXEC sp_addrolemember 'db_datawriter', 'EdFiOdsAdminApp'
+EXEC sp_addrolemember 'db_owner', 'EdFiOdsAdminApp'
+
+```
+
+**Create EdFiOdsProductionApi user login** Expand source
+
+```sql
+--For creating EdFiOdsProductionApi login
+use [master]
+IF NOT EXISTS(SELECT name FROM sys.sql_logins WHERE name='EdFiOdsProductionApi')
+BEGIN
+CREATE LOGIN [EdFiOdsProductionApi] WITH PASSWORD = 'user provided password'
+END;
+
+--Create database user for EdFiOdsProductionApi login on EdFi_Admin database
+use [EdFi_Admin]
+IF DATABASE_PRINCIPAL_ID('EdFiOdsProductionApi') IS NULL
+BEGIN
+    CREATE USER [EdFiOdsProductionApi] FOR LOGIN [EdFiOdsProductionApi] WITH DEFAULT_SCHEMA=[dbo]
+END
+-- Assign roles to EdFiOdsProductionApi on EdFi_Admin database
+EXEC sp_addrolemember 'db_datareader', 'EdFiOdsProductionApi'
+EXEC sp_addrolemember 'db_datawriter', 'EdFiOdsProductionApi'
+IF DATABASE_PRINCIPAL_ID('db_executor') IS NULL
+BEGIN
+    CREATE ROLE [db_executor]
+END
+GRANT EXECUTE TO [db_executor]
+EXEC sp_addrolemember 'db_executor', 'EdFiOdsProductionApi'
+
+--Create database user for EdFiOdsProductionApi login on EdFi_Ods_Production database
+use [EdFi_Ods_Production]
+IF DATABASE_PRINCIPAL_ID('EdFiOdsProductionApi') IS NULL
+BEGIN
+    CREATE USER [EdFiOdsProductionApi] FOR LOGIN [EdFiOdsProductionApi] WITH DEFAULT_SCHEMA=[dbo]
+END
+--Assign roles to EdFiOdsProductionApi on EdFi_Ods_Production database
+EXEC sp_addrolemember 'db_datareader', 'EdFiOdsProductionApi'
+EXEC sp_addrolemember 'db_datawriter', 'EdFiOdsProductionApi'
+IF DATABASE_PRINCIPAL_ID('db_executor') IS NULL
+BEGIN
+    CREATE ROLE [db_executor]
+END
+GRANT EXECUTE TO [db_executor]
+EXEC sp_addrolemember 'db_executor', 'EdFiOdsProductionApi'
+
+--Create database user for EdFiOdsProductionApi login on EdFi_Security database
+use [EdFi_Security]
+IF DATABASE_PRINCIPAL_ID('EdFiOdsProductionApi') IS NULL
+BEGIN
+    CREATE USER [EdFiOdsProductionApi] FOR LOGIN [EdFiOdsProductionApi] WITH DEFAULT_SCHEMA=[dbo]
+END
+--Assign roles to EdFiOdsAdminApp on EdFi_Security database
+EXEC sp_addrolemember 'db_datareader', 'EdFiOdsProductionApi'
+EXEC sp_addrolemember 'db_datawriter', 'EdFiOdsProductionApi'
+
+```
+
+6\. Once EdFiOdsProductionApi user login creation done, now user should be able
+to launch the ODS API website successful on Azure.
+
+![Launcher exe](https://edfidocs.blob.core.windows.net/$web/img/reference/admin-app/technical-articles/image2021-5-14_17-49-15.png)
+
+7. Please use above mentioned Admin app upgrade steps for installing Admin App
+2.2.0.
+
+8\. Please refere Admin App new features in Admin App
+2.2.0 [here](https://edfi.atlassian.net/wiki/display/ADMIN/Next+Steps).
```

Comment:
Please update this and look for other opportunities to replace links back to Confluence.

If you find a link to Ed-Fi ODS/API older than version 7, then you can leave that pointing to Confluence, since we haven't moved the older documentation yet.

Reply:
link removed